src/utils.ts

```diff
@@ -3,13 +3,17 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 export function isObject(v: unknown): v is Record<string | number | symbol, unknown> {
-  return (v && typeof v === "object" && v.constructor === Object);
+  return !!(v && typeof v === "object" && v.constructor === Object);
```

Comment:
```suggestion
  return (!!v && typeof v === "object" && v.constructor === Object);
```

Is this what you're really looking for? Forcing truthiness outside makes it a bit harder to understand it on the first look otherwise.

Reply:
No, so I had to add this `!!` because typescript was giving me an error `Type 'unknown' is not assignable to type 'boolean'.`. I searched this, but this seems to be the accepted workaround to coerce to a boolean, TS does not do that on its own. Another option would be to do this `<boolean>(v && typeof v === "object" && v.constructor === Object);` I have no strong opinions, if you think the second option is clearer I can change it.

Reply:
What I needed to do with this change, was coerce the whole expression to a boolean, not just `v`.

Reply:
Okay, I was attending to your review comments and I added an extra test to the storage suite that I realized was missing. Which was for the best because I found a bug in this utils functions (I'll file a bug to add tests specifically to this module, I should have done that from the start). 

Anyways, this works: `(typeof v === "boolean" || (typeof v === "object" && v !== null && v.constructor === Boolean))`. I need to check if `v` is `null` **after** I check that it is of type object. This happens because `null` is of type object and TS apparently only takes into account the last condition that you set when asserting the possible type of something.